core/src/main/java/org/everit/json/schema/ValidationFailureReporter.java

```diff
@@ -37,4 +37,12 @@ ValidationException inContextOfSchema(Schema schema, Runnable task) {
     }
 
     abstract void validationFinished();
+
+    public Object getState() {
+        return null;
+    }
+
+    public boolean isChanged(Object oldState) {
```

Comment:
Please let's keep these new methods (`#getState()` and `#isChanged()`) package-private. These are implementation-specific details and not public APIs.

Reply:
Done